### PR TITLE
Add TPG

### PR DESCRIPTION
**Pull request by fletcherdodd**

### CODE REVIEW

File: entries/t/tpg.com.au.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "TPG": {
+    "domain": "tpg.com.au",
+    "tfa": [
+      "sms"
+    ],
+    "documentation": "https://support.tpg.com.au/multi-factor-authentication-tpg",
```
Comment (hkamran80) [inline entries/t/tpg.com.au.json:7]: I'm getting an HTTP 403 Forbidden response to the entire `support.tpg.com.au` subdomain. Is it restricted to Australia or am I blocked for some reason by some CDN?

Comment (Carlgo11) [inline entries/t/tpg.com.au.json:7]: Could you take a screenshot? They seem to have geo-blocked me 😅

Comment (hkamran80) [reply entries/t/tpg.com.au.json:7]: We reviewed at the same time again! 😅

Comment (fletcherdodd) [reply entries/t/tpg.com.au.json:7]: image

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Pull request merged**